src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -122,21 +122,39 @@ public RoutingResponse route() {
                 firstRemovedItinerary,
                 filteredItineraries,
                 routingErrors,
-                debugTimingAggregator,
-                reverseFilteringDirection
+                debugTimingAggregator
         );
     }
 
     /**
      * Adjust the 'dateTime' if the page cursor is set to "goto next page".
      * The date-time is used for many things, for example finding the days to search,
      * but the transit search is using the cursor[if exist], not the date-time.
+     *
+     *          * The {@code dateTime}, {@code arriveBy} and {@code pageCursor} depend on each other, to
+     *          * enforce the integrity of the theses three variables, we set them all with this method.
+     *          *
+     *          * @param dateTime The earliest-departure-time {@code arriveBy=false} or latest-arrival-time
+     *          *                 {@code arriveBy=true} for this search. If {@code null} now is used.
+     *          * @param pageCursor The pageCursor will override the dateTime.
```

Comment:
Indentation

Reply:
This method is moved to the Request ([feature-envy](https://refactoring.guru/smells/feature-envy)), where the doc is fixed.